plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/CliOptions.java

```diff
@@ -283,7 +286,27 @@ private List<Plugin> getPlugins() {
                 throw new PluginInputException("Unknown file type, file must have .yaml/.yml or .txt extension");
             }
         }
-        return requestedPlugins;
+         List<String> pluginNames = new ArrayList<>();
```

Comment:
This seems like it is generating a list of plugin file names rather than plugin names.  Would it be better to rename the variable `pluginFileNames` for clarity?
```suggestion
         List<String> pluginNames = new ArrayList<>();
```

Reply:
Done @MarkEWaite